### DISCUSSION TITLE
Expose extra network config

### DISCUSSION THREAD

**Discussion by solarisfire** [mysticaltech/terraform-hcloud-kube-hetzner#1526] [Ideas]
### Description

Could extra config be exposed for network configuration?

I run my network with a vSwitch Connected, all routes exposed to the vSwitch, and another subnet defined, however I have no way to configure this within this module, and the vSwitch config gets destroyed with each terraform apply.

**Comment by mysticaltech** [comment]
@solarisfire It's a good idea that we will consider for sure. PRs always welcome!

**Comment by solarisfire** [comment]
Sure, I'm not sure I understand the stack well enough to code it myself, would need to figure things out.

It also becomes a problem on terraform destroy as the network cannot be deleted by terraform if a vSwitch has been manually connected and it's route's exposed.

Will try and find time at some point.
